starting_package/openvino_workbench/docker_config_creator.py

```diff
@@ -148,6 +148,7 @@ def create_config_for_container(passed_arguments: Namespace) -> dict:
                               'PYTHON_WRAPPER': 1},
               'name': passed_arguments.container_name,
               'ports': {INTERNAL_PORT: (passed_arguments.ip, passed_arguments.port)},
+              'hostname': 'workbench',
```

Comment:
Need to check this parameter

Reply:
Are you done with checks?

Reply:
Yes, everything pass well with this parameter.